packages/terra-date-time-picker/tests/wdio/date-time-picker-spec.js

```diff
@@ -212,6 +212,27 @@ describe('DateTimePicker', () => {
     Terra.should.matchScreenshot('1', { viewports });
   });
 
+  describe('OnSelect', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-date-time-picker/date-time-picker/date-time-picker-on-select');
+      browser.click('[class*="button"]');
+    });
+
+    Terra.should.matchScreenshot('0', { viewports });
```

Comment:
The "tiny" OnSelect[0].png screenshot does not match the large screenshot. Should they?

Reply:
A similar thing happens between some of the other screenshots so I assumed that this was expected behavior. Probably gonna poke around to double check though.
Excluded_Dates_are_Disabled[default].png for instance
Large https://github.com/cerner/terra-framework/blob/master/packages/terra-date-time-picker/tests/wdio/__snapshots__/reference/en/chrome_large/date-time-picker-spec/Excluded_Dates_are_Disabled%5Bdefault%5D.png
Tiny https://github.com/cerner/terra-framework/blob/master/packages/terra-date-time-picker/tests/wdio/__snapshots__/reference/en/chrome_tiny/date-time-picker-spec/Excluded_Dates_are_Disabled%5Bdefault%5D.png

Reply:
Hmm actually, I think this test should be using beforeEach() instead of before

Reply:
Yep that was it according to @benbcai, I can just remove the {viewports} argument and we should be okay.

Reply:
Fixed here https://github.com/cerner/terra-framework/pull/585/commits/bf7770d8eb04e4701794086433d6b1f6f1609afe
Specifically I reworked the tests. Since the purpose of the tests was to show that we start with no "onSelect" or "onChange" values and how we set these values, I have changed the tests to not show the calendar at all in `tiny` view so that it matches `large` view.